scripts/west_commands/runners/nrf_common.py

```diff
@@ -383,7 +383,7 @@ def do_require(self):
         ''' Ensure the tool is installed '''
 
     def op_program(self, hex_file, erase, qspi_erase, defer=False, core=None):
-        args = {'firmware': {'file': hex_file, 'format': 'NRFDL_FW_INTEL_HEX'},
+        args = {'firmware': {'file': hex_file},
```

Comment:
have you tried if nrfjprog keeps working after this change?

Reply:
I just successfully flashed nrf52840dk/nrf52840 with `west flash`. I don't see how `nrfjprog` would be broken by this, especially that https://github.com/zephyrproject-rtos/zephyr/blob/53ddff639562ef68dc0a6f62b82f7505c75ebdce/scripts/west_commands/runners/nrfjprog.py#L63-L83 doesn't use `_op['firmware']['format'])`)

Reply:
thanks, just wanted to confirm as this common file is shared